src/main/java/reactor/netty/http/client/HttpClientDoOnError.java

```diff
@@ -82,7 +82,11 @@ protected TcpClient tcpConfiguration() {
 			this.headers = c.headers;
 			this.cookieDecoder = c.cookieDecoder;
 			this.uri = c.uri == null ? c.uriStr : c.uri.toString();
-			this.path = HttpOperations.resolvePath(this.uri);
+			if(this.uri != null){ 
+				this.path = HttpOperations.resolvePath(this.uri);
+			}else{
+				this.path = null;
+			}
```

Comment:
Can we simplify that `this.path = this.uri != null ? HttpOperations.resolvePath(this.uri) : null;`

Reply:
Has been modified